demo-app/src/main/java/com/jfcbuilder/demo/JFreeChartBuilderDemo.java

```diff
@@ -447,7 +452,9 @@ private static ChartBuilder getDailyStockChartBuilder(StockChartConfig cfg) {
         .annotation(XYBoxBuilder.get()
           .x1(p3Date).y1(70.0)
           .x2(p4Date).y2(30.0)
-          .outlineStyle(THICK_DASHED_LINE));
+          .outlineStyle(THICK_DASHED_LINE))
+        
+        .annotation(XYTitleBuilder.get().title(new TextTitle("Indicator Title")).x(0.1).y(1.0));
```

Comment:
Move the x-coordinate to the left edge (0.0) for sample code on working with the boundary.